backend/src/routes/bookmark.ts

```diff
@@ -145,4 +145,29 @@ bookmarkRouter.delete("/:id", async (c) => {
     c.status(500);
     return c.json({ error: "Something went wrong " });
   }
-});
\ No newline at end of file
+});
+
+bookmarkRouter.get("/bookmarkCount",async(c)=>{
+  const prisma = new PrismaClient({
+    datasourceUrl: c.env.DATABASE_URL,
+  }).$extends(withAccelerate());
+  const userId = c.get("userId");
+  try {
+    const count = await prisma.bookmark.count({
+      where:{
+        userId:userId
+      }
+    });
+    if(!count){
+      throw Error("Error Finding Count")
+    }
+    return c.json({
+      count:count
+    })
+  } catch (error:any) {
+      c.status(400);
+      return c.json({
+        "error":`${error.message}`
+      })
+  }
+})
```

Comment:
Instead of creating a new bookmark route to get the bookmark count, you can accommodate bookmark property in the existing `/api/v1/user/:userId` route.
This will reduce the number of API calls we are making from the frontend.